api/client/project.go

```diff
@@ -55,6 +55,78 @@ func (p *Padl) CreateProject(name, description string, bits int) (*padlfile.File
 	return &pf, nil
 }
 
+//CreateDeployKey Creates a new DeployKey
+func (p *Padl) CreateDeployKey(projectName string, keyName string) (*payloads.CreateDeployKeyResponse, error) {
+	pl := &payloads.CreateDeployKeyRequest{
+		DeployKeyName: keyName,
+	}
+	plBytes, err := json.Marshal(&pl)
+	if err != nil {
+		return nil, fmt.Errorf("could not marshall payload: %s", err)
+	}
+	req, err := http.NewRequest(
+		http.MethodPost,
+		fmt.Sprintf("%s/project/%s/deploy_key", p.HostURL, projectName),
+		bytes.NewBuffer(plBytes))
+	if err != nil {
+		return nil, fmt.Errorf("could not build http requests: %s", err)
+	}
+	p.setAuth(req)
+
+	resp, err := p.HTTPClient.Do(req)
+	if err != nil {
+		return nil, fmt.Errorf("could not send http request: %s", err)
+	}
+
+	respByt, err := ioutil.ReadAll(resp.Body)
+	defer resp.Body.Close()
+	if err != nil {
+		return nil, fmt.Errorf("could not read http response body: %s", err)
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		fmt.Println(string(respByt))
```

Comment:
rm this, we should eventually make a general catch-all for all the 400's at least. i.e. printErrIf4xx(resp)